design/design-doc-stacks.md

```diff
@@ -195,42 +200,69 @@ spec:
-* `resources` directory: This directory contains all the CRDs and optional metadata about them.  These CRDs are the types that the custom controller implements the logic for.  They will be directly installed into Crossplane so that users can create instances of them to start consuming their new Stack functionality.
-* `ui-schema.yaml`: UI metadata that will be transformed and annotated according to the [Stack UI Metadata One Pager](one-pager-stack-ui-metadata.md)
+* `icon.svg`: This file (or `icon.png`, `icon.jpg`, `icon.gif`, or potentially other supported filenames, TBD) will be used in a visual context when listing or describing this stack.  The preferred formats/filenames are `svg`, `png`, `jpg`, `gif` in that order (if multiple files exist).  For bitmap formats, the width to height ratio should be 1:1. Limitations may be placed on the acceptable file dimensions and byte size (TBD).
+* `resources` directory: This directory contains all the CRDs and optional metadata about them.
+  * `group.yaml`: Related Stack resources (CRDs) can be described at a high level within a group directory using this file.
+  * `resource.yaml` and `icon.svg`: Files that describe the resource with descriptions, titles, or images, may be used to inform out-of-cluster or in-cluster Stack managing tools.  This may affect the annotations of the `Extension` record or the Resource CRD (TBD).
+  * `ui-schema.yaml`: UI metadata that will be transformed and annotated according to the [Stack UI Metadata One Pager](one-pager-stack-ui-metadata.md)
+  * `*crd.yaml`: These CRDs are the types that the custom controller implements the logic for.  They will be directly installed into Crossplane so that users can create instances of them to start consuming their new Stack functionality.  Notice that the filenames can vary from `very.descriptive.name.crd.yaml` to `crd.yaml`.
+  Multiple CRDs can reside in the same file.  These CRDs may also be pre-annotated at build time with annotations describing the `resource.yaml`, `icon.svg`, and `ui-schema.yaml` files to avoid bundling additional files and incurring a minor processing penalty at runtime.
+
+The minimum required file tree for a single tool, such as the Stack Manager, could be condensed to the following:
+
+```text
+.registry/
+├── app.yaml
+└── resources
+      └── crd.yaml
+```
+
+## Example Package Files
+
+A concrete examples of this package format can be examined at <https://github.com/crossplaneio/sample-extension>.
+
+A Git repository may choose to include the `.registry` directory with all of the files described above but that may not always be the case.  Stacks are easy to create as a build artifacts through a combination of shell scripting, Make, Docker, or any other tool-chain or process that can produce an OCI image.  Another example project that uses processes to convert the artifacts of Kubebuilder 2 to create a Stack is available at <>.
```

Comment:
Is the end of this paragraph intending to link to https://github.com/crossplaneio/sample-stack-wordpress?

Reply:
Oi. Yes.